docs/source/telegram.games-tree.rst

```diff
@@ -1,6 +1,19 @@
 Games
 -----
 
+Your bot can offer users **HTML5 games** to play solo or to compete against each other in groups and one-on-one chats. Create games via `@BotFather <https://telegram.me/BotFather>`_ using the /*newgame* command. Please note that this kind of power requires responsibility: you will need to accept the terms for each game that your bots will be offering.
```

Comment:
`/*newgame*` will not render correctly. Please use <code>\`\`/newgame\`\`</code> to ensure that it's rendered as mono-spaced text.